### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by UNDERHMA**
Slight changes to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,12 +50,10 @@ Run `ng serve` and navigate to `http://localhost:4200/` to make sure the applica
...
+In order to log into the application, you will need to create an account (DO NOT USE ACTUAL LOGIN CREDENTIALS YOU NORMALLY USE) when you login to the front end on localhost:4200 for the first time. This will create a user in the account on Google Firebase. Firebase manages the users and we call Firebase to get a token for a valid user on the front end when authentication occurs. Then, when we make a request to the backend, the backend only accepts requests with a valid token from the firebase environment. The JSON information in the "firebase-service-credentials.json" allows our back-end to verify valid JWTs for the Firebase instance William created.  The information in the "environment.ts" file allows our front-end to authenticate with the Firebase instance.
```
Comment (RevatureGentry) [inline README.md]: Are you referring to the Service Account when you mention `firebase-service-credentials.json`? If so, please put an additional note here that this file should NEVER be added to version control and to reach out to the Center of Excellence if the current iteration has any further questions about where it is, how to use it, etc.

**Review by RevatureGentry — CHANGES_REQUESTED** [review]
Please add more detail to the Service Account information for the next iteration.

**Comment by UNDERHMA** [comment]
Yes, the Service Account information is what I am referring to, just has to be that name based on what the Java inputstream that loads it looks for. Do the most recent changes I made to the file provide adequate warning? If we want a more complete overview of the Service Account information the backend uses, I would suggest we add that to the readme for the backend.

**Comment by UNDERHMA** [comment]
Sorry, read your first note to quickly. Added the note about reaching out to the Center of Excellence.

**Review by RevatureGentry — APPROVED** [review]

**Pull request merged**